lib/omnibus-flapjack/publish.rb

```diff
@@ -198,6 +199,28 @@ def add_to_rpm_repo(pkg, component = 'experimental')
           Mixlib::ShellOut.new('createrepo . 2>&1', :live_stream => $stdout).run_command.error!
         end
       end
+
+      def add_to_packagecloud(pkg, component='experimental')
+        begin
+          credentials = Packagecloud::Credentials.new(ENV['packagecloud_user'], ENV['packagecloud_token'])
```

Comment:
It'd be neater to have the credentials picked up from env vars (or wherever) in the Rakefile and then passed through to OmnibusFlapjack::Publish#add_to_packagecloud using an options hash. One reason is to make it easier to test the code.